### DISCUSSION TITLE
Timeline for Pimcore X?

### DISCUSSION THREAD

**Discussion by Cruiser13** [pimcore/pimcore#15228] [General]
Now that Pimcore 11 is available I wonder what the timeline for Pimcore X is. Will we get security updates for the next 3 months (as we had for Pimcore 6)?

**Comment by jdreesen** [comment]
Hopefully way longer :wink:

**Reply by aarongerig** [reply]
Would also love to see something similar to this: https://typo3.org/cms/roadmap#section-support-times 😊

**Comment by fashxp** [comment]
Pimcore X (10.6) will be supported in CE for at least 3 months inkl. bugfixes, eventually longer. We will communicate that in the course of next week. 
After that period, there will be our LTS offering for security updates. 

There is already a short overview [here](https://github.com/pimcore/pimcore/#readme), we will update and extend that accordingly.

**Comment by fashxp** [comment]
We have some updates regarding EOL and planned release dates for Pimcore 10: 

- **Last Pimcore 10.5 bugfix release** will be 10.5.24 on **June 21st 2023**, which means also EOL for Pimcore 10.5. 
  - To keep that date, we have already PR-freeze for 10.5 branch.
  - We will review and potentially merge existing PRs for 10.5 branch though. 

- **Last Pimcore 10.6 bugfix release** will be on **August 16th 2023**, which means also EOL for Pimcore 10.6 and Pimcore 10 in general. 
  - Until then, we will release 10.6 bugfix version on a regular base. 
  - PR-freeze will be about 1 week in advance. 

After that period, there will be our **commercial LTS offering** for security updates for **Pimcore 10**.

**Reply by Cruiser13** [reply]
Thank you @fashxp for the information. We hoped for more time for the migration but now we can make a plan. Thanks!

**Comment by Cruiser13** [comment]
@fashxp will we still get the 10.6.8 release? It should contain a security fix: https://github.com/pimcore/pimcore/pull/15752

**Reply by dvesh3** [reply]
@Cruiser13 about to tag the release 😉

**Reply by Cruiser13** [reply]
Awesome, thanks!
